controllers/TuitController.ts

```diff
@@ -166,9 +166,16 @@ export default class TuitController implements ITuitController {
    * @param {Response} res Represents response to client, array of all the versions of the 
    * Tuit.
    */
-  getVersions = (req: Request, res: Response) =>{
-    TuitController.tuitVersionDao.findAllPreviousVersions(req.params.tid)
-    .then((tuitVersion: TuitVersion[]) => res.json(tuitVersion))
+  getVersions = async (req: Request, res: Response) =>{
+    const currentTuit = await TuitController.tuitDao.findTuitById(req.params.tid)
+    // TuitController.tuitVersionDao.findAllPreviousVersions(req.params.tid)
+    // .then((tuitVersion: TuitVersion[]) => res.json(tuitVersion))
+    const currentTuitVersion = {"_id":currentTuit._id,"tuit":currentTuit.tuit,
+        "v":currentTuit.v,"editedOn":currentTuit.postedOn}
+    const prevTuitVersion = await TuitController.tuitVersionDao.findAllPreviousVersions(req.params.tid)
+    var allVersions = Object.assign({},currentTuitVersion,prevTuitVersion)
+    return(res.json(allVersions))
+    
```

Comment:
Let me know if there's a better way to go about this method. Attaching a screenshot of current response:
<img width="855" alt="Screen Shot 2022-11-30 at 6 23 11 PM" src="https://user-images.githubusercontent.com/38109555/204929354-caa28731-f75a-4a01-b0f9-c3fce3baed8f.png">

Reply:
This works!